student/pom.xml

```diff
@@ -312,4 +314,39 @@
 		</dependency>
 
 	</dependencies>
+
+	<build>
+		<plugins>
+			<plugin>
+				<groupId>com.spotify</groupId>
+				<artifactId>docker-maven-plugin</artifactId>
+				<version>0.4.10</version>
+				<configuration>
+					<imageName>${docker.image.prefix}/${project.artifactId}</imageName>
+					<imageTags>
+						<imageTag>${project.version}</imageTag>
+						<imageTag>latest</imageTag>
+					</imageTags>
+					<dockerDirectory>${project.basedir}/src/main/docker</dockerDirectory>
+					<resources>
+						<resource>
+							<targetPath>/</targetPath>
+							<directory>${project.build.directory}</directory>
+							<include>${project.build.finalName}.war</include>
+						</resource>
+
+						<!-- TDS_Student requires file-system test definitions.  The resource directories below must
+							be defined in the build machine's local file-system at the specified paths
+							to be embedded into the student docker container image.
+							Modify as required.-->
+						<resource>
+							<targetPath>/</targetPath>
+							<directory>/usr/local/tomcat/resources/tds/bank</directory>
+							<include>**/*</include>
+						</resource>
```

Comment:
This adds the "item" and "stimuli" directories containing test data to the docker directory so they can be embedded into the container image.